src/components/_common/Button/Button.tsx

```diff
@@ -1,3 +1,73 @@
-export const Button = ({ label }: { label: string }) => {
-  return <button>{label}</button>;
+import styled from '@emotion/styled';
+import {
+  BRIGHT_GREY,
+  PRIMARY_BLUE,
+  PRIMARY_BLUE_HOVER,
+  SECONDARY_NAVY,
+  SECONDARY_NAVY_HOVER,
+} from '@/style/colorConstants';
+
+interface ButtonProps {
+  color: 'BLUE' | 'NAVY';
+  isOutlineButton: boolean;
```

Comment:
```suggestion
  isOutline: boolean;
```
Button은 빼도 괜찮을 것 같습니다!